src/app/pages/quickMaps/pages/dietaryChange/components/comparisonCard/scenariosMap/scenariosMap.component.ts

```diff
@@ -98,45 +94,23 @@ export class ScenariosMapComponent implements AfterViewInit {
 
   private initialiseListeners(): void {
     this.baselineMap.on({
-      mousedown: () => {
-        this.allowBaselineMapEvents = true;
-        this.allowScenarioMapEvents = false;
-      },
       move: () => {
-        if (this.allowBaselineMapEvents) {
-          this.setMapPosition(this.baselineMap, this.scenarioMap);
-        }
-      },
-      zoom: () => {
-        this.setMapZoom(this.baselineMap, this.scenarioMap);
+        this.alignMaps(this.baselineMap, this.scenarioMap);
       },
     });
     this.scenarioMap.on({
-      mousedown: () => {
-        this.allowScenarioMapEvents = true;
-        this.allowBaselineMapEvents = false;
-      },
       move: () => {
-        if (this.allowScenarioMapEvents) {
-          this.setMapPosition(this.scenarioMap, this.baselineMap);
-        }
-      },
-      zoom: () => {
-        this.setMapZoom(this.scenarioMap, this.baselineMap);
+        this.alignMaps(this.scenarioMap, this.baselineMap);
       },
     });
   }
 
-  private setMapPosition(baseMap: L.Map, targetMap: L.Map): void {
-    if (targetMap.getCenter() !== baseMap.getCenter()) {
-      targetMap.panTo(baseMap.getCenter());
-    }
-  }
-
-  private setMapZoom(baseMap: L.Map, targetMap: L.Map): void {
-    if (targetMap.getZoom() !== baseMap.getZoom()) {
-      targetMap.setZoom(baseMap.getZoom());
-    }
+  private alignMaps(baseMap: L.Map, targetMap: L.Map): void {
```

Comment:
You might want to change the name of the variable here as "baseMap" is a bit too similar to "baseline". 
I dunno what to :-)

primary and secondary?
control and mirror?
changing and repeating?